### DISCUSSION TITLE
[OP Stack Deployment] Node not syncing after restart

### DISCUSSION THREAD

**Discussion by 0x366** [ethereum-optimism/developers#365] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Sorry, didnt know about discussions and opened an issue - https://github.com/ethereum-optimism/optimism/issues/10267

**Comment by hash1go** [comment]
No problem!
If you'd like to continue the discussion in the issue, feel free to close this discussion.
